ipn/local.go

```diff
@@ -369,6 +369,8 @@ func (b *LocalBackend) updateFilter(netMap *controlclient.NetworkMap) {
 	}
 }
 
+var WantPorts = []uint16{22, 80, 443, 3389, 5900, 8000, 8080, 8443, 8888}
```

Comment:
Why is this exported?

Also it seems very arbitrary, at least the 8xxxx ones.

I really still don't like this whitelist approach.